metricbeat/docs/modules/redis.asciidoc

```diff
@@ -63,9 +63,9 @@ metricbeat.modules:
   #maxconn: 10
 
   # Filters can be used to reduce the number of fields sent.
-  #filters:
+  #processors:
   #  - include_fields:
-  #      fields: ["stats"]
+  #      fields: ["beat", "metricset", "redis.info.stats"]
```

Comment:
This is quite a breaking change i think. Meaning if someone use `fields: ["abc"]` before and keeps the config, it will now remove many more fields.